lib/block_view_flush.go

```diff
@@ -140,6 +140,24 @@ func (bav *UtxoView) FlushToDbWithTxn(txn *badger.Txn, blockHeight uint64) error
 	if err := bav._flushNonceEntriesToDbWithTxn(txn); err != nil {
 		return err
 	}
+	if err := bav._flushValidatorEntriesToDbWithTxn(txn, blockHeight); err != nil {
+		return err
+	}
+	if err := bav._flushGlobalStakeAmountNanosToDbWithTxn(txn, blockHeight); err != nil {
+		return err
+	}
+	if err := bav._flushStakeEntriesToDbWithTxn(txn, blockHeight); err != nil {
+		return err
+	}
+	if err := bav._flushLockedStakeEntriesToDbWithTxn(txn, blockHeight); err != nil {
+		return err
+	}
+	// TODO: We may want to move this into a new FlushToDb function that only flushes
+	// entries set in the OnEpochEndHook. No sense in wasting a bunch of cycles flushing
+	// all the other entries which will always be nil/empty in the OnEpochEndHook.
```

Comment:
Note to self: What would be the point of this? I think the flush is just a noop if we're not at the block height.